docs/how-to/websites-on-ipfs/link-a-domain.md

```diff
@@ -8,11 +8,32 @@ In this guide you will learn how to link up an existing domain name to the CID o
 
 ## DNS
 
+To get started you'll need:
+
+- A domain name registrar
+- A website.
+- An idea for a domain name.
+
+Access your registrar's control panel to manage the CNAME record and TXT record.
+
+CNAME: `www.gateway.ipfs.io.`
+
+TXT: `dnslink=/ipfs/$SITE_CID`
+
+![](./images/link-a-domain/dns-add-content-record.png)
+
+Save and in a few hours, you'll be able to go to www.YourDomainName.com and view your website.
```

Comment:
```suggestion
Save your changes. Traditional domain names can take some time to propagate through the internet, so it may take a few hours for your domain to point to your site hosted on IPFS.
```